src/main/java/com/taf/business/api/Request.java

```diff
@@ -0,0 +1,24 @@
+package com.taf.business.api;
+
+import io.restassured.http.Method;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+@Data
+@NoArgsConstructor
+public class Request {
+
+    private String requestUri;
+    private Method method;
+    private Map<String, ?> queryParams;
+    private String body;
+
+    public void clean() {
+        requestUri = null;
+        method = null;
+        queryParams = null;
+        body = null;
+    }
```

Comment:
I'd propose to reset Request object reference to null somewhere in the code instead of resetting each field in object itself

Reply:
Fixed after code refactoring.